sensu_plugin/handler.py

```diff
@@ -20,19 +20,19 @@
     from urlparse import urlparse
 except ImportError:
     from urllib.parse import urlparse
-from sensu_plugin.utils import get_settings
+from utils import get_settings
```

Comment:
I've tested this locally as we discussed on slack and `from sensu_plugin.utils import get_settings` works perfectly for me, the issues that caused you to need to make this change were probably due to the way you were deploying the package.